docs/pipeline_configuration.md

```diff
@@ -0,0 +1,75 @@
+## Overview
```

Comment:
I would expect the main header to be "Pipeline configuration files" if you ask me.